### PR TITLE
Refactor medusa/*.py files

### PR DESCRIPTION
**Pull request by labrys**
- [X] PR is based on the DEVELOP branch
- [X] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [X] Read [contribution guide](https://github.com/PyMedusa/SickRage/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/exception_handler.py [hunk 1]
```diff
@@ -17,9 +17,9 @@ def handle(thread_name, err):
     """
     if isinstance(err, OSError):
         if err.errno == 28:
-            logger.warning('Out of disk space: {error_msg}', error_msg=err.message)
+            logger.warning('Out of disk space: {error_msg}', error_msg=err)
```
Comment (fernandog) [inline medusa/exception_handler.py]: @ratoaq2 fyi

File: medusa/history.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 import db
 from .common import FAILED, Quality, SNATCHED, SUBTITLED
 from .helper.encoding import ss
-from .show.History import History
+from .show.history import History
```
Comment (fernandog) [inline medusa/history.py]: @labrys i would import as `show_history` to not confuse with history.py. maybe also rename show/show_history.py

Comment (labrys) [inline medusa/history.py]: `from .show.history` already differentiates between `show/history.py` and `history.py`.  If `history.py` also had a `class History:` then we could import it as ShowHistory to differentiate, but thats not the case here.

**Comment by labrys** [comment]
Closing this as #1402 makes the conflict resolution more of a pain than just redoing the refactors.

**Pull request closed without merging**